static/usage/v7/radio/using-comparewith/angular/example_component_html.md

```diff
@@ -0,0 +1,9 @@
+```html
+<ion-list>
+  <ion-radio-group [compareWith]="compareWith" (ionChange)="handleChange($event)">
+    <ion-item *ngFor="let food of foods">
```

Comment:
Can we use a trackBy function here? https://ionicframework.com/docs/angular/performance#ngfor-with-ionic-components

(It's similar to the `key` usage in your React example)